StripeConnect/StripeConnectTests/WebView+Tests.swift

```diff
@@ -135,23 +137,23 @@ extension WKWebView {
     @discardableResult
     func evaluateMessageWithReply(name: String,
                                   json: String,
-                                  postReply: Bool = true,
-                                  completionHandler: ((Any?, (any Error)?) -> Void)? = nil) async throws -> Any? {
+                                  verifyResult: ((Data) -> Void)? = nil) async throws -> Any? {
```

Comment:
Why does `verifyResult` need to be a `Data` type instead of decoding to the Response type like it was before?

Reply:
Because the send messages are no longer decodable. Previously I had everything set as codable, but Appearance can't be Codable.

Reply:
ah, that makes sense. I like your recent change though of comparing the JSON string instead, so I think that addresses this issue.